snuba/migrations/snuba_migrations/spans_v2/0001_spans_v2.py

```diff
@@ -0,0 +1,82 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, Float, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+    Column("trace_id", UUID()),
+    Column("transaction_span_id", UInt(64)),
+    Column("span_id", UInt(64)),
+    Column("parent_span_id", UInt(64, Modifiers(nullable=True))),
+    Column("transaction_name", String()),
+    Column("span_group_id", UUID()),
+    Column("op", String()),
+    Column("status", UInt(8)),
+    # transaction start/finish times
+    Column("start_ts", DateTime()),
+    Column("start_ns", UInt(32)),
+    Column("finish_ts", DateTime()),
+    Column("finish_ns", UInt(32)),
+    # span start/finish times
+    Column("span_start_ts", DateTime()),
+    Column("span_start_ns", UInt(32)),
+    Column("span_finish_ts", DateTime()),
+    Column("span_finish_ns", UInt(32)),
+    Column("duration_ms", UInt(32)),
+    Column("exclusive_time_ms", Float(64)),
+    Column("retention_days", UInt(16)),
+    Column("deleted", UInt(8)),
+]
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.CreateTable(
+                storage_set=StorageSetKey.TRANSACTIONS,
+                table_name="spans_v2_local",
+                columns=columns,
+                engine=table_engines.ReplacingMergeTree(
+                    storage_set=StorageSetKey.TRANSACTIONS,
+                    version_column="deleted",
+                    order_by="(project_id, toStartOfDay(finish_ts), transaction_name, cityHash64(transaction_span_id)), cityHash64(span_id)",
```

Comment:
The order by here starts the same way as the [order by on the transactions table](https://github.com/getsentry/snuba/blob/a0d15215b4c017c4f1d02c385b39ff5a9177db73/snuba/migrations/snuba_migrations/transactions/0001_transactions.py#L73). The differences are
- `span_id` on the transactions table is `transaction_span_id` on the span table
- the span table is additionally sorting by the individual span's `span_id`